tests/util/fixture_fluent.py

```diff
@@ -4,87 +4,119 @@
 from ansys.fluent.core.examples import download_file
 
 
-@pytest.fixture
-def exhaust_system_geometry():
-    global _exhaust_system_geometry_filename
-    if not _exhaust_system_geometry_filename:
-        _exhaust_system_geometry_filename = download_file(
-            filename="exhaust_system.fmd", directory="pyfluent/exhaust_system"
+def get_file_type(full_file_name):
+    if ".msh" in full_file_name:
+        return "mesh"
+    if ".cas" in full_file_name:
+        return "case"
+    if ".dat" in full_file_name:
+        return "data"
+    if (
+        ".fmd" in full_file_name
+        or ".scdoc" in full_file_name
+        or ".pmdb" in full_file_name
```

Comment:
from pathlib import Path
ext = Path(full_file_name).suffix
if ext in (".fmd", ".scdoc", ".pmdb"):
etc.